tests/test_endtoend.py

```diff
@@ -25,3 +25,17 @@ def test_prefix_augment(self):
 		with open('contrib/test-data/test_zhen_config_prefix.expected.out', 'r', encoding='utf-8') as reffile:
 			reference: str = "".join(reffile.readlines())
 		self.assertEqual(output, reference)
+
+	def test_no_shuffle(self):
+		output: str = subprocess.check_output([sys.executable, '-m', 'opustrainer', '-c', 'contrib/test_enzh_config_plain.yml', '-d', '-n'], encoding="utf-8")
+		reference: str = ""
+		with open('contrib/test-data/clean.enzh.10', 'r', encoding='utf-8') as reffile:
+			reference: str = "".join(reffile.readlines())
+		# Since we read 100 lines at a time, we wrap. Often.
+		# Hence, for the test to pass we need to read the number of lines in the test file
+		reference_arr = reference.split('\n')
```

Comment:
Why this join + split instead of:
```py
with open(...) as reffile:
  reference_arr = list(line.rstrip('\r\n') for line in reffile)
```

Reply:
I wanted to keep it as similar as the above options

Reply:
Aah I missed that context. Let's keep it like you had it then!